src/components/experiences/negative-experiences.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import styled from 'styled-components';  
+
+import { DialogContent, DialogActions } from '@material-ui/core';
+import ProgressBottom from '@general/navs/progress-bottom'; 
+
+import PrimaryButton from '@general/buttons/primary-button';  
+
+import {ReactComponent as HeartIcon} from '@assets/images/heart-icon.svg';
+
+export default function NegativeExperiences(props) {    
+ 
+
+    function showNextButton() { 
+      if(props.payload.tags.length === 0) { 
+        return false;
+      } else { 
+        return true;
+      }
+    } 
+
+    function handleSelection(tag) { 
+      let arr = props.payload.tags;
+
+      if(arr.includes(tag)){ 
+        arr = arr.filter( (item) => (!(item === tag)) );
+        props.handler({
+          edited: props.payload.edited, 
+          positive: props.payload.positive, 
+          tags: arr
+        });
+      } else { 
+        arr.push(tag);
+        props.handler({
+          edited: props.payload.edited, 
+          positive: props.payload.positive, 
+          tags: arr
+        });
+      }
+    }  
+
+    return (
+    <> 
+    <Content>
+        What made your testing experience Negative?
+    </Content>    
+     <Actions>  
+      <Box>
+      <TagButton color={'primary'} variant={props.payload.tags.includes('ph') ? 'contained' : 'outlined'} onClick={() => handleSelection('ph')}> 
```

Comment:
this should be handled with a for loop and a config array instead of hardcoding every option